pkg/virtctl/root.go

```diff
@@ -68,6 +69,8 @@ func NewVirtctlCommand() *cobra.Command {
 	rootCmd.AddCommand(
 		console.NewCommand(clientConfig),
 		vnc.NewCommand(clientConfig),
+		ssh.NewCommand(clientConfig),
+		ssh.NewLoadTestCommand(clientConfig),
```

Comment:
i'm hesitant for this load test to be within virtctl. Is there a precedence for kubectl providing any sort of load testing support like this? My expectation is that this laod test would be it's own binary.  It might even belong as a functional test case that is skipped by default, but can be be run with a `focus` to execute against a cluster.

Reply:
Yeah, I just wanted to avoid creating a new tool just for this and the easiest was to add it here and instead I made it a hidden command, so it shouldn't show up to the unknowing user (as far as I know).

Reply:
I can remove it, just got to keep the code around somewhere for (probably) myself so if something comes up I still can do the tests.